accumulo-store/src/main/java/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -139,8 +140,9 @@ public Properties getPropertiesFromValue(final String group, final Value value)
             return properties;
         }
         final MapWritable map = new MapWritable();
-        try {
-            map.readFields(new DataInputStream(new ByteArrayInputStream(value.get())));
+        try (final InputStream inStream = new ByteArrayInputStream(value.get());
+             final DataInputStream dataStream = new DataInputStream(inStream)) {
+            map.readFields(dataStream);
```

Comment:
Why not add a finally block to these try catches and close these resources there? In this case the streams aren't needed once the objects are created, this could be added in a few of the changed files.